cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Poll.hs

```diff
@@ -9,27 +9,28 @@ import           Cardano.Api.Shelley
 
 import           Data.Text (Text)
 
-data GovernancePollCmds era
+data GovernancePollCmds
   = GovernanceCreatePoll -- ^ Create a SPO poll
-      (BabbageEraOnwards era) {- TODO smelc, use BabbageEraOnly here instead -}
+      -- (BabbageEraOnwards era) -- TODO Use when legacy command is removed
```

Comment:
We should keep this and change it to `BabbageEraOnly` as per the original TODO

Reply:
Kept this :+1: